src/content/structured/components/select/code.mdx

```diff
@@ -447,9 +447,9 @@ export const SelectExampleRecommended = () => (
 export const snippetsValidation = [
```

Comment:
This code snippet doesn't match the component preview. There should be a warning and error select in these snippets as well as the success.